### PR TITLE
Added icon for dev environment

### PR DESCRIPTION
**Pull request by yair100**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- For #6933
- Related #issue...

**Details of Changes**
Add details of changes here.
- Added...
- Fixed...

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@BanCrash this is ready now

**Comment by BanCrash** [comment]
@yair100 did it work the powershell script for you? I have to use this modified line in order to replace the Assets:
`(Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline`
(The change is that I've added an extra backslash on Assets\AppTilesDev because without that it wasn't working for me)

Also, is there any way to test within the pipeline? I manually tested using the powershell script modifying the `$(Build.SourcesDirectory)\src` part in order to get it to work in my local machine but I don't know if there is any way to test it directly from the pipeline.

File: builds/azure-pipelines-release.yml [hunk 1]
```diff
@@ -26,12 +26,22 @@ jobs:
...
         $xmlDoc.Save('$(Build.SourcesDirectory)\src\Files.Package\Package.appxmanifest')
       failOnStderr: true
 
+  - task: PowerShell@2
+    displayName: 'Use Release Logo'
+    inputs:
+      targetType: 'inline'
+      script: |
+        gci $(Build.SourcesDirectory)\src -Include *.csproj, *.appxmanifest, *.wapproj -recurse | ForEach -Process {
+        (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```
Comment (BanCrash) [inline builds/azure-pipelines-release.yml:41]: Regarding my previous comment, this is the thing that I had to change in order to work. (1/3)
```suggestion
        (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```

File: builds/azure-pipelines-release.yml [hunk 2]
```diff
@@ -117,6 +127,16 @@ jobs:
...
```
Comment (BanCrash) [inline builds/azure-pipelines-release.yml:136]: (2/3)
```suggestion
        (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```

File: builds/azure-pipelines.yml [hunk 3]
```diff
@@ -17,12 +17,22 @@ steps:
...
       $xmlDoc.Save('$(Build.SourcesDirectory)\src\Files.Package\Package.appxmanifest')
     failOnStderr: true
 
+- task: PowerShell@2
+  displayName: 'Use Release Logo'
+  inputs:
+    targetType: 'inline'
+    script: |
+      gci $(Build.SourcesDirectory)\src -Include *.csproj, *.appxmanifest, *.wapproj -recurse | ForEach -Process {
+      (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```
Comment (BanCrash) [inline builds/azure-pipelines.yml:32]: (3/3)
```suggestion
      (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```

**Comment by yair100** [comment]
> @yair100 did it work the powershell script for you? I have to use this modified line in order to replace the Assets: `(Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline` (The change is that I've added an extra backslash on Assets\AppTilesDev because without that it wasn't working for me)
> 
> Also, is there any way to test within the pipeline? I manually tested using the powershell script modifying the `$(Build.SourcesDirectory)\src` part in order to get it to work in my local machine but I don't know if there is any way to test it directly from the pipeline.

@BanCrash the easiest way to test is to download the artifact from the pipeline and extract the package to inspect the assets.

**Review by BanCrash — APPROVED** [review]
> @BanCrash the easiest way to test is to download the artifact from the pipeline and extract the package to inspect the assets.

@yaichenbaum Thanks! I did that and confirmed that the assets are replaced.

LGTM

**Pull request merged**